spec/models/article_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 RSpec.describe Article, type: :model do
```

Comment:
You could also test that your factory works. example: "is valid when using factory"